Documentation/adding-new-repository.md

```diff
@@ -69,7 +69,7 @@ Please use templates as much as possible. The standard is a bit different depend
 1. Modify Mender QA Pipeline:
   * ref https://github.com/mendersoftware/mender-qa/blob/master/.gitlab-ci.yml
   * Add new repo into `variables`
-  * Add new call to `checkout_repo` in `init_workspace`.
+  * Add new call to `checkout_repo` in `init:workspace`.
```

Comment:
To be honest, I am unsure about this one. There is only one job in init, so I'll rather call it `init`, `init-workspace`, `set-workspace` or whatever, but when using `init:` prefix it sounds to me like there should be other parallel jobs in init (which is not the case)

Reply:
Sure, I wasn't sure either. Dropping the commit!